src/main/java/com/til/socialapp/controller/MainController.java

```diff
@@ -105,11 +105,11 @@ public Tag updateTag(@RequestBody Tag t1)
 		return ret;
 	}
 	@GetMapping("/common/fetch")
-	public TeamDep addTeamDepService()
+	public Common addTeamDepService()
```

Comment:
Its okay to have a class to return the tags and teams but please store these separately instead of in a single Document. 

Have a separate collection to store master Teams (includes designation and departments), like we discussed last time and keep the master tags in a separate collection